src/datasources/asset/data-sources/asset-summary/AssetSummaryDataSource.ts

```diff
@@ -24,8 +24,8 @@ export class AssetSummaryDataSource extends AssetDataSourceBase {
         return this.processSummaryQuery(query as AssetQuery);
     }
 
-    shouldRunQuery(_: AssetQuery): boolean {
-        return true;
+    shouldRunQuery(query: AssetQuery): boolean {
+        return !query?.hide;
```

Comment:
@OliverCosma do you remember why optional chaining was used here? Are there scenarios where `query` was `undefined` that you encountered? I'm trying to make the pattern consistent in #456 and opted to use the pattern shown here consistently but was not sure if that was necessary.